Allow to disable writing computation times (#1385)
sphinx-gallery/sphinx-gallery: doc/configuration.rst

duration of scripts that ran faster than that amount will not be logged nor
embedded in the html output.

.. _write_computation_times:

Write computation times
=======================

> Set to ``false`` if you want to omit computation times from the output.

Reviewer: ```suggestion
Set to ``False`` if you want to omit computation times from the output.
```

> Set to ``false`` if you want to omit computation times from the output.

Reviewer 2: ```suggestion
Set to ``False`` if you want to omit computation times from all gallery outputs.
```

This helps with reproducible builds.

> If the ``SOURCE_DATE_EPOCH`` environment variable is set, it defaults to false.
> True otherwise.

Reviewer: ```suggestion
Default is ``True`` unless the ``SOURCE_DATE_EPOCH`` environment variable is set.
```
